src/Interactive/Features/Interactive/Core/InteractiveHost.Service.cs

```diff
@@ -426,22 +426,15 @@ private static void ReportUnhandledException(Exception e)
 
             #region Operations
 
-            public ObjectFormattingOptions ObjectFormattingOptions
+            // TODO (tomat): testing only
```

Comment:
Doesn't work. This is called remotely and is required to be public. Will kill remoting eventually anyways.